week-8/Homework/mandatory/3-dog-photo-gallery/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Getting Dog images From API</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div id="content">
+      <ul id="uList">
+        <li id="list"><img id="image" /></li>
```

Comment:
This works, but on the first load of the page, before clicking the button, it looks a little weird :) Can you think of a way to make the page not have a strange black blob displayed?

Reply:
Actually, sorry, I just re-read the task - it should be _adding_ an image each time you click, rather than replacing - can you give that a go? :)

Reply:
Thanks, Daniel I have done it mate.thanks again.